bootstrap.Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG GOLANG_IMAGE=golang:1.17.5-alpine@sha256:4918412049183afe42f1ecaf8f5c2a88917c2eab153ce5ecf4bf2d55c1507b74
-ARG NIX_IMAGE=nixos/nix:2.6.0@sha256:f085a81ca39642a405b41f45e8c21a369d89c65ca9f05d3a9b78dc5852d300ff
+ARG GOLANG_IMAGE=docker.io/golang:1.17.5-alpine@sha256:4918412049183afe42f1ecaf8f5c2a88917c2eab153ce5ecf4bf2d55c1507b74
+ARG NIX_IMAGE=docker.io/nixos/nix:2.11.0@sha256:ef91f7bb5ef25d95b8a0437ce0ee5d1de5cfdf8ac882f1597f785068ea1e28af
```

Comment:
Any reason to use full name?

Reply:
some systems don't assume docker.io as default

Reply:
Out of curiosity, what systems? It seems to me that at this point the ship has sailed and refs like this should be interpreted as `docker.io` for any system to be compatible with the world. As awkward as it is to couple tooling to particular hosts.